spinalcordtoolbox/reports/qc2.py

```diff
@@ -779,8 +790,14 @@ def mosaic(img: Image, centers: np.ndarray, radius: tuple[int, int] = (15, 15),
 
     If `img` has N slices, then `centers` should have shape (N, 2).
     """
-    # Fit as many slices as possible in each row of 600 pixels
-    num_col = math.floor(600 / (2*radius[0]*scale))
+    # Note: This function used to hardcode a max row width of 600 pixels
+    #       In practice, because the canvas size is fixed to 1500 pixels, this
+    #       resulted in a permanent upscaling by 2.5x when saving the image.
+    #       To make things clearer, we now use a variable.
+    max_row_width = TARGET_WIDTH_PIXL / scale
```

Comment:
The main idea with making this change is that:

- Instead of scaling _up_ each mosaic image by an integer factor, we can
- Scale _down_ the array size using `scale`, keep the mosaic images a 1:1 pixel representation, and let `matplotlib` handle the upscaling.